frontend/views/containers/settings/UserProfile.vue

```diff
@@ -170,43 +152,47 @@ export default {
       sbp('okTurtles.events/emit', OPEN_MODAL, mode)
       return false
     },
-    async save () {
-      if (this.ephemeral.newPicture) {
-        try {
-          this.form.picture = await imageUpload(this.form.picture)
-          this.ephemeral.newPicture = false
-        } catch (error) {
-          console.error(error)
-          this.ephemeral.errorMsg = L('Failed to upload user picture')
-          return false
+    async fileChange (fileList) {
+      if (!fileList.length) return
+      const fileReceived = fileList[0]
+      let picture
+
+      this.$refs.picture.setFromBlob(fileReceived)
+
+      try {
+        picture = await imageUpload(fileReceived)
+      } catch (error) {
+        console.error(error)
+        this.$refs.pictureMsg.danger(`${L('Failed to upload avatar. Please, try again.')} ${error.message}`)
```

Comment:
`${L('Failed to upload avatar. Please, try again.')} ${error.message}` => `L('Error uploading avatar: {errorMsg}', { errorMsg: error.message })`

Reply:
Once again, I think @mmbotelho has a word on this :)

Reply:
@mmbotelho is welcome to comment, but my [response](https://github.com/okTurtles/group-income-simple/pull/779#discussion_r350262715) remains the same.

Reply:
Can you give me examples of{errorMsg} @taoeffect @sandrina-p ?

Will the user need to know the specific error? Or can we keep it simple?

Reply:
errorMsg is literally the code error, for example "Error 500 - not found" or "can't find 'expired' of undefined" or any other JS internal error that can happen.

Reply:
That's good to include in the message. It will help the user help us help them.

So I am for including `{errorMsg}`. If both of you are against it, then please replace `{errorMsg}` with at least: `Please check the {link_}error logs{_link} for more info.` Where `{link_}` is used to go to the debug page mentioned in issue #595 (that doesn't exist yet)

Reply:
This appears to have been missed, please double-check that all error messages and calls to `.danger` are consistent and don't use `${` stuff